ping.go

```diff
@@ -1,3 +1,4 @@
+// Deprecated: This package is no longer maintained.
 // Package ping is a simple but powerful ICMP echo (ping) library.
 //
```

Comment:
```suggestion
// Package ping is a simple but powerful ICMP echo (ping) library.
//
// Deprecated: This package is no longer maintained.
```

Reply:
I added it just above `package ping` as your previous comment. What to do?

Reply:
Either works according to the [Go docs](https://go.dev/wiki/Deprecated#examples). Up to you.